sync/sync.go

```diff
@@ -122,7 +131,9 @@ func (s *Synchronizer) syncBlocks(syncCtx context.Context) {
 	verifiers := stream.New().WithMaxGoroutines(runtime.NumCPU())
 
 	streamCtx, streamCancel := context.WithCancel(syncCtx)
+
 	nextHeight := s.nextHeight()
+	StartingBlockNumber = &nextHeight
```

Comment:
`nextHeight` constantly changes. So `StartingBlockNumber` will change as well, since it is just a pointer to `nextHeight` . Is stil intentional?

Reply:
No, it isn't intentional. I've fixed it now.